packages/mask-markup/src/components/correct-input.jsx

```diff
@@ -43,8 +46,9 @@ export default withStyles(() => ({
     borderColor: color.correct()
   }
 }))(props => {
-  const { correct, isBox, classes, disabled, ...rest } = props;
+  const { correct, isBox, classes, disabled, maxLength, ...rest } = props;
```

Comment:
Please check if this is not used by other components. If it's used, make sure to support that version as well (to not break anything)